src/components/product-set.js

```diff
@@ -171,12 +168,12 @@ export default class ProductSet extends Component {
   }
 
   /**
-   * make request to SDK for current page + 1 to determine if next page exists. Render button if next page exists.
+   * make request to SDK for next page. Render button if products on next page exist.
    * @return {Promise} promise resolving when button is rendered or not.
    */
   showPagination() {
-    return this.sdkFetch({page: this.page + 1}).then((data) => {
-      this.nextModel = {products: data};
+    return this.props.client.fetchNextPage(this.model.products[this.model.products.length - 1]).then((data) => {
```

Comment:
it might be more terse to just say `fetchNextPage(this.model.products)`

Reply:
Updated!